packages/cf-component-select/test/Select.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { shallow } from 'enzyme';
 import toJson from 'enzyme-to-json';
-import Select from '../../cf-component-select/src/index';
+import { Select } from '../../cf-component-select/src/index';
 
 test('should render', () => {
   const tree = shallow(<Select />);
```

Comment:
`mount()` would be better. 

1. the snapshots contains only the Fela wrapper
2. `shallow()` doesn't pass context (theme) down correctly